README.md

```diff
@@ -1,37 +1,93 @@
-# The Bone Yard -- frontend
+# The Bone Yard -- Frontend
 
-As a dog owner, I'd like to be able to set-up "doggie play-dates" with other owners and dogs but there's no safe and functional way to do that. This app attempts to tackle that monumentous problem.
 
-The app [(which can be found here)](https://the-bone-yard-fe.herokuapp.com/#/) currently allows a user to search for dog parks in their current area, or a specific area of their choosing.
+## Project Description 
 
-## Purpose
+This application is dog park locator that allows users to search for dog parks near them or to search by city and state. Users can view details about a particular dog park, get driving directions to a specific dog park, and save dog parks they are interested in. This is a full stack application with the front-end housed on this repository. The repository for the back-end can be found [here](https://github.com/the-bone-yard/backend-repo). This project is the capstone project completed during the final module of the Front End Engineering program at the Turing School of Software and Design. 
 
-This repo serves as the main frontend portion of the application. The backend can be found [here.](https://github.com/the-bone-yard/backend-repo)
+The BoneYard deployed application can be found [HERE.](https://the-bone-yard-fe.herokuapp.com/#/)
 
 ## Contributors
 
 Bret Merritt
 [<img align="left" alt="Bret's github" width="22px" src="https://raw.githubusercontent.com/iconic/open-iconic/master/svg/globe.svg" />][git-bret]
 [<img align="left" alt="Bret's linkedin' | LinkedIn" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/linkedin.svg" />][linkedin-bret]
-<br />
-<br>
+
 Nathan Darrington
 [<img align="left" alt="Nathan's github" width="22px" src="https://raw.githubusercontent.com/iconic/open-iconic/master/svg/globe.svg" />][git-nath]
 [<img align="left" alt="Nathan's linkedin' | LinkedIn" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/linkedin.svg" />][linkedin-nath]
-<br />
-<br>
+
 Nicole Gooden
 [<img align="left" alt="Nicole's github" width="22px" src="https://raw.githubusercontent.com/iconic/open-iconic/master/svg/globe.svg" />][git-nicole]
 [<img align="left" alt="Nicole's linkedin' | LinkedIn" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/linkedin.svg" />][linkedin-nicole]
-<br />
-<br>
+
 Stacy Potten
 [<img align="left" alt="Stacy's github" width="22px" src="https://raw.githubusercontent.com/iconic/open-iconic/master/svg/globe.svg" />][git-stac]
 [<img align="left" alt="Stacy's linkedin' | LinkedIn" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/linkedin.svg" />][linkedin-stac]
-<br />
+
+
+## Front-End Technologies
+
+- JavaScript
+- Vue
+- GoogleMaps API
+- Jest
+- Babel
+- HTML5
+- CSS/SCSS
+- Travis CI
+- Heroku
+
 
 ## Set-up
 
+The deployed application can be found [HERE.](https://the-bone-yard-fe.herokuapp.com/#/) 
```

Comment:
You can remove line 44 since this same line exists in the Project Description.  :)